src/preact/component/3p-frame.js

```diff
@@ -114,17 +125,17 @@ function ProxyIframeEmbedWithRef(
       ),
       src,
     });
-  }, [count, nameProp, srcProp, title, type]);
+  }, [contextOptions, count, nameProp, options, srcProp, title, type]);
 
   return (
     <IframeEmbed
-      allow={BLOCK_SYNC_XHR}
+      allow={allow}
       contentRef={contentRef}
       messageHandler={messageHandler}
       name={name}
       ref={ref}
       ready={!!name}
-      sandbox={sandbox}
+      sandbox={excludeSandbox ? undefined : sandbox}
```

Comment:
Sigh, I was going to suggest using `sandbox={null}`, but it seems Preact will incorrectly coerce that to an empty string.

Reply:
Should we file an issue with Preact?

Reply:
Yes, if the behaviour is different than other JSX syntax environments.